core/mybatis-generator-core/src/main/java/org/mybatis/generator/internal/util/ClassloaderUtility.java

```diff
@@ -47,8 +51,8 @@ public static ClassLoader getCustomClassloader(Collection<String> entries) {
             for (String classPathEntry : entries) {
                 file = new File(classPathEntry);
                 if (!file.exists()) {
-                    throw new RuntimeException(getString(
-                            "RuntimeError.9", classPathEntry)); //$NON-NLS-1$
+                    LOG.warn("classPathEntry " + classPathEntry + " not exists!" );
+                    continue;
```

Comment:
Please change this to follow the pattern of using an externalized string

Reply:
OK， I tried slf4j2 pattern.
LOG.warn("classPathEntry {} not exists", classPathentry);
not work.
Can you give me some demo?